corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -83,27 +85,28 @@ public CorfuStoreBrowser(CorfuRuntime runtime) {
     public CorfuStoreBrowser(CorfuRuntime runtime, String diskPath) {
         this.runtime = runtime;
         this.diskPath = diskPath;
+        dynamicProtobufSerializer =
+            new DynamicProtobufSerializer(runtime);
+        Serializers.registerSerializer(dynamicProtobufSerializer);
     }
 
-    /**
-     * Validate that the namespace is not null
-     * @param namespace
-     */
-    private static void verifyNamespace(String namespace) {
-        if (namespace == null) {
-            throw new IllegalArgumentException("Please specify --namespace");
-        }
+    private boolean isEmptyOrNull(String str) {
+        return (str == null || str.isEmpty());
```

Comment:
Not needed. See comment https://github.com/CorfuDB/CorfuDB/pull/2969/files#r663237174